stack/stack.py

```diff
@@ -8,18 +8,45 @@
    as the underlying storage structure.
    Make sure the Stack tests pass.
 3. What is the difference between using an array vs. a linked list when 
-   implementing a Stack?
+   implementing a Stack? Where using an array and performing these built in methods causes 
+    the computer to keep creating and deleting whole arrays to perform the tasks.
 """
+# class Stack:
+#     def __init__(self):
+#         self.size = 0
+#         self.storage = []
+
+#     def __len__(self):
+#         return self.size
+#         pass
+
+#     def push(self, value):
+#         self.size += 1
+#         self.storage.insert(0, value)
+#         pass
+
+#     def pop(self):
+#         if self.size > 0:
+#             self.size = self.size - 1
+#             return self.storage.pop(0)
```

Comment:
Similar to insert, popping from the beginning? or the end? Which is more efficient?